nginx/conf.d/server/server.account

```diff
@@ -4,6 +4,15 @@ include /etc/nginx/conf.d/include/ssl-settings;
 include /etc/nginx/conf.d/include/init-optional-variables;
 
 location / {
+    rewrite_by_lua_block {
+        local skynet_account = require("skynet.account")
+
+        if skynet_account.accounts_disabled() and skynet_account.should_redirect_for_disabled_accounts() then
+            local utils = require("utils")
+            return ngx.redirect(utils.getenv("DISABLED_ACCOUNTS_REDIRECT_URL"))
```

Comment:
For our current needs (`account.siasky.net` -> `siasky.net/siasky-account-notice`) this could be a 301, but I left it default (302) as personally I see more use cases for this variable (e.g. having a custom page for when maintenance is happening on accounts service).